impl/src/test/java/org/ehcache/PersistentCacheManagerTest.java

```diff
@@ -21,24 +21,26 @@
 import org.junit.Test;
 
 import java.io.File;
+import java.io.IOException;
 
 import static org.ehcache.CacheManagerBuilder.newCacheManagerBuilder;
 import static org.hamcrest.CoreMatchers.instanceOf;
 import static org.hamcrest.CoreMatchers.is;
 import static org.junit.Assert.assertThat;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
+import org.junit.Rule;
+import org.junit.rules.TemporaryFolder;
 
 /**
  * @author Alex Snaps
  */
 public class PersistentCacheManagerTest {
 
+  @Rule
+  public final TemporaryFolder folder = new TemporaryFolder();
+  
   @Test
-  public void testInitializesLocalPersistenceService() {
-    final File rootDirectory = mock(File.class);
-    when(rootDirectory.exists()).thenReturn(true);
-    when(rootDirectory.getAbsolutePath()).thenReturn("CRAP!");
+  public void testInitializesLocalPersistenceService() throws IOException {
+    final File rootDirectory = folder.newFolder("testInitializesLocalPersistenceService");
```

Comment:
With these changes, the test should fail as no exception is thrown anymore. The only reason this still passes is because there is no `fail` call before the catch block. Can you adapt the test to make sure it tests what was tested before?